lib/envars.ml

```diff
@@ -209,7 +209,8 @@ let print_config ?(prefix_var_name="") f =
   let open Printf in
   fprintf f "%sCOQLIB=%s/\n" prefix_var_name (coqlib ());
   fprintf f "%sCOQCORELIB=%s/\n" prefix_var_name (coqlib () / "../coq-core/");
-  fprintf f "%sDOCDIR=%s/\n" prefix_var_name (docdir ());
+  (* We keep installing doc for Coq Makefile under $DOCDIR/coq *)
+  fprintf f "%sDOCDIR=%s/coq/\n" prefix_var_name (docdir ());
```

Comment:
Is this coq_makefile-specific code? I'm worried about the confusion between `DOCDIR` in `Makefile.doc` that does not contain `/coq` and `DOCDIR` here that does. Wouldn't it just make more sense to introduce a `DOCDIRDEST` variable in coq_makefile that adds the `/coq` part like in `Makefile.doc`?

Reply:
This print_config function is used in coq_makefile and for -config, so it's mostly coq_makefile specific (not sure if there are any -config users).

Reply:
We could put the `/coq/` part in https://github.com/coq/coq/blob/b0b91acf55ac613bcb589d5b9bd966390ca6a483/tools/CoqMakefile.in#L172 (only non-debug use of docdir)

Reply:
Good idea, done.